hera_cal/tests/test_firstcal.py

```diff
@@ -313,3 +313,20 @@ def test_single_file_execution_nocalfile(self):
         nt.assert_true(os.path.exists(objective_file))
         os.remove(objective_file)
         return
+
+    def test_overwrite(self):
+        objective_file = os.path.join(
```

Comment:
In my understanding, this test will remove the objective file if it exists, make a new empty file, and runs firstcal with the `--overwrite` flag to make an actual calfits file. However, the test just checks that the file exists, which would be true if the test silently failed (for whatever reason). I think a more precise test is to open and read the newly written file to ensure it's a calfits file, and verify one or two of the file properties are what we expect it to be. I know this is a higher bar than many of the firstcal tests typically have (though incidentally we should probably be more stringently testing the output of other tests too). It's just that in this case, we're purposely putting a file where we're eventually going to go looking for one, so assuming nothing throws an error, this test will pass.

Reply:
good point!